src/components/Announcements.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import styled from 'styled-components'
 import { format } from 'timeago.js'
 import ReactMarkdown from 'react-markdown'
-import { Card } from './Common'
+import { Card, maxWidthMediaQueries } from './Common'
```

Comment:
Thoughts on moving the media queries into the theme? So they are accessible like `${p => p.theme.breakpoints.xs}` etc?